Automattic/woocommerce-payments
# In-person payments: return non-empty account structures only

Fixes 991-gh-Automattic/woocommerce-payments-server

#### Changes proposed in this Pull Request

- Instead of returning `[]` causing errors tracking spam in mobile apps, return non-empty data structure.

#### Testing instructions

- Modify `includes/admin/class-wc-rest-payments-accounts-controller.php` file and replace `'permission_callback' => [ $this, 'check_permission' ]` with `'permission_callback' => '__return_true'`
- Execute `curl -X GET {store_url}/wp-json/wc/v3/payments/accounts`
- Ensure the endpoint is not broken
- UTs will cover the rest of testing

-------------------

- [ ] Close the loop in pdfdoF-1U-p2
